lib/auth/apikey.ex

```diff
@@ -9,58 +12,77 @@ defmodule Auth.Apikey do
   schema "apikeys" do
     field :client_secret, :binary
     field :client_id, :binary
-    field :description, :string
-    field :name, :string
-    field :url, :binary
     field :person_id, :id
     field :status, :id
+    belongs_to :app, Auth.App
 
     timestamps()
   end
 
-  @doc false
+  @doc """
```

Comment:
These functions aren't new. I just moved them over from the `ApikeysController` file which was deleted in this PR. 👍